### PR TITLE
Add a Mathics notebook service (with test)

### PR DESCRIPTION
**Pull request by benley**

### CODE REVIEW

File: nixos/modules/services/misc/mathics.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+with lib;
+
+let
+  cfg = config.services.mathics;
+
+in {
+  options = {
+    services.mathics = {
+      enable = mkEnableOption "Autostart a Mathics notebook service.";
```
Comment (jagajaga) [inline nixos/modules/services/misc/mathics.nix]: Change argument to `Mathics notebook service` please.

Comment (benley) [inline nixos/modules/services/misc/mathics.nix]: Done!

**Comment by benley** [comment]
sorry for the long delay - this should be ready for another look now

File: nixos/modules/services/misc/mathics.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+with lib;
+
+let
+  cfg = config.services.mathics;
+
+in {
+  options = {
+    services.mathics = {
+      enable = mkEnableOption "Mathics notebook service.";
```
Comment (jagajaga) [inline nixos/modules/services/misc/mathics.nix]: Please remove dot at the end :)

Comment (benley) [inline nixos/modules/services/misc/mathics.nix]: lol - done :)

**Pull request merged**